Resources/Prototypes/Traits/physical.yml

```diff
@@ -231,3 +231,127 @@
           Blunt: 1.5
           Slash: 1.5
           Piercing: 1.5
+
+- type: trait
+  id: Small
+  category: Physical
+  points: -2
+  requirements:
+    - !type:CharacterTraitRequirement
+      inverted: true
+      traits:
+        - Tiny
+    - !type:CharacterSpeciesRequirement
+      inverted: true
+      species:
+        - Felinid
+    - !type:CharacterHeightRequirement
+      max: 150
+    - !type:CharacterWidthRequirement
+      max: 32
+  components:
+    - type: PseudoItem
+      storedOffset: 0,17
+      shape:
+        - 0,0,1,4
+        - 0,2,3,4
+        - 4,0,5,4
+
+- type: trait
+  id: Tiny
+  category: Physical
+  points: -3
+  requirements:
+    - !type:CharacterTraitRequirement
+      inverted: true
+      traits:
+        - Small
+    - !type:CharacterHeightRequirement
+      max: 133
+    - !type:CharacterWidthRequirement
+      max: 29
+  componentRemovals:
+    - PseudoItem
+  components:
+    - type: Item
+      size: Normal
```

Comment:
ARE YOU SURE ABOUT THIS?

Reply:
This seem somehow OP and... really annoying to just grab peoples.
I mean i seen it in high roleplays but with toggles.

Even still, this coulld make a ton of isssues regarding antags/combat.

Reply:
Yes.

Reply:
Counterpoint, you can put people in the deep fryer who take this trait.
You can also shove them into your pocket. Open an airlock. Then Ctrl-Q them into space.